### DISCUSSION TITLE
Python example import is not resolved

### DISCUSSION THREAD

**Discussion by invm** [TypeFox/monaco-languageclient#692] [Q&A]
Hi there!

Getting this issue when running the python example
<img width="675" alt="image" src="https://github.com/TypeFox/monaco-languageclient/assets/52990657/ba58a95b-fb3a-48a3-8139-42195b622807">

But the lsp seems to recognize the file.
<img width="581" alt="image" src="https://github.com/TypeFox/monaco-languageclient/assets/52990657/ce016143-fdb3-40c6-95ee-7302620d64ce">

Am I missing something?
How can I open the seconds file with `Go To File` action or navigate with `Go to Defenition`

**Comment by kaisalmen** [comment]
Hi @invm please see this comment: https://github.com/TypeFox/monaco-languageclient/issues/653#issuecomment-2095370533
You can overcome that problem, but there is no general solution, yet.
